src/App.tsx

```diff
@@ -8,7 +8,38 @@ import { TodoFilter } from './components/TodoFilter';
 import { TodoModal } from './components/TodoModal';
 import { Loader } from './components/Loader';
 
+import { getTodos } from './api';
+import { Todo } from './types/Todo';
+
 export const App: React.FC = () => {
+  const [arrOfTodos, setTodos] = useState<Todo[]>([]);
+  const [sortParams, setSortParam] = useState({
+    sortSelect: '',
+    sortInput: '',
+  });
+  const [selectedTodo, setTodo] = useState<Todo | null>(null);
+
+  const setFilterParam = (select: string, input: string) => {
+    setSortParam({
+      sortSelect: select,
+      sortInput: input,
+    });
+  };
+
+  const setSelectedTodo = (todo: Todo | null) => {
+    setTodo(todo);
+  };
+
+  useEffect(() => {
+    const fetchData = async () => {
+      const arrayTodosFromServer = await getTodos();
```

Comment:
handle errors and loading process

Reply:
I don't really understand what you mean by 'handle loading process', so I only handled errors.

Reply:
if you do not understand something - you should ask about it in the Slack channel to avoid repeated PR rejection